### ISSUE TITLE
Internal announcement 

### ISSUE THREAD

**Issue by arianaescobar** [CartoDB/carto-vl#326]
We should communicate the project internally so everyone in the company is aligned with what this project is, it's value proposition, and so on before CARTO Locations NY. 

- Event? (Like a town hall, demo day, or in an ad-hoc special event to explain it) 
- Email?
- Slides?
- Project doc? 

cc @rochoa @inigomedina @namessanti

**Comment by inigomedina** [comment]
Yes. We have the Product Updates for that. Today we will communicate Auth, CARTO.js v4, and the Dev Center in there. So, we can make the same with CARTO VL.

**Comment by arianaescobar** [comment]
RIGHT! Duh. I completely forgot, sorry 😓

**Comment by namessanti** [comment]
Here's the [communication plan](https://docs.google.com/spreadsheets/d/1BwjxRGzikRSOxRz7mV0PuHX9H06_8UMwuo25TSV9Uh4/edit#gid=0) from PMM (Row 16).

As you can see we need to think about a few things:
* Set a date for full sales training (Santiago + SME training on the what, why, and the value prop) Who's the best SME to work with me? @makella @arianaescobar @inigomedina - What about Tuesday May 15th? A lot of this will also come from/be int andem with the Press Release scheduled May 23rd.
* Technical training for solutions (Led by SME) Who's the best SME to lead this? Looking at Wednesday May 16th.
* Internal Email - @inigomedina this probably requires it's own product email with the what, why, value prop, and sales messaging baked in

There's also a bunch of external comms you can see in the spreadsheet we'll need to create from this core material.

**Comment by arianaescobar** [comment]
Thanks, @namessanti. 

Please, let's break down what we need to do on separate issues so we can track and assign everything separately.

I already created issues for your [first](https://github.com/CartoDB/carto-vl/issues/319) and [second point](https://github.com/CartoDB/carto-vl/issues/320). So we can follow the conversation about those there. 

For the internal announcement, if I'm understanding correctly, we'll do a: 

- [ ] Blog post on the Product Announcements blog; and 
- [ ] Email for team@carto.com with the what, why, value prop, and sales messaging.

Anything else for that? 

Please, create on this repo any other issue for those external comms if you're going to need help from us so we can add it to our kanban 🙂
